sql/01-create-table-student.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE student_info.student(
+  id INT PRIMARY KEY NOT NULL,
+  name CHAR(20) NOT NULL,
+  grade INT
+);
```

Comment:
When creating a table it is best practice not to use dot notation for the table name. This table could be called student and would work fine. I would also not recommend usage of "not null" in this situation. Below is an example of how I completed the problem (in my example I used Serial, which auto increments the same way saying "Primary Key" would. Serial is specific to PostgreSQL so typing out Primary Key like you did might actually be better)

`CREATE TABLE student (
 id SERIAL PRIMARY KEY,
 name TEXT,
 grade INTEGER
 );`